cmd/cli/plugin/package/package_available_get.go

```diff
@@ -132,6 +132,9 @@ func getValuesSchema(cmd *cobra.Command, args []string, kc kappclient.Client) er
 	}
 
 	var parseErr error
+	if len(pkg.Spec.ValuesSchema.OpenAPIv3.Raw) == 0 {
```

Comment:
Could you please add a unit-test to cover this case.

Reply:
unit tests will be added by https://github.com/vmware-tanzu/tanzu-framework/pull/1056

Reply:
suggestion: `ValuesSchema` and `OpenAPIv3` are optional fields and they are both nullable, that means `pkg.Spec.ValuesSchema` or `pkg.Spec.ValuesSchema.OpenAPIv3` could be `nil`. We might want to add more checks from line#135